controllers/etcd_controller.go

```diff
@@ -179,6 +181,9 @@ func (r *EtcdReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	if !reflect.DeepEqual(etcd.Spec, etcdCopy.Spec) {
 		etcdCopy.Spec = etcd.Spec
 	}
+
+	etcdCopy.Status.ObserverdGeneration = &etcdCopy.Generation
```

Comment:
Where is the update-status call for this?

Reply:
The idea is to eventually update this in `updateEtcdErrorStatus` or `updateEtcdStatus`.

Reply:
OK, I see. Thanks.